data/process_json.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import json
+
+inp_term="term_to_cui.json"  #input terms/cui json file
+defs="cui_to_def.json"
+s_data="jargon.json" 		#secondary data set to combine
+termF="term_to_cui_final.json"		#final output json file
+defF="cui_to_def_final.json"
+
+def load_files(inp_file):
+	with open(inp_file, 'r') as f:		#opens and loads terms/cui
+		loaded_file=json.load(f)
+	f.close()
+	return loaded_file
+	
+def main():
+	terms_cui=load_files(inp_term)		#input terms/cui json file
+	sec_data=load_files(s_data)			#MedJ and Medline Plus set
+	def_data=load_files(defs)			#cui to defintions
+	
+	new_terms_cui={}
+	terms_removed=0
+	term_count=0
+	
+	for k in terms_cui.keys():
+		if ("+" in k) or (">" in k) or ("\\" in k) or ("kit kit" in k) or ("(" in k) or ("&" in k) or ("_" in k) or (":" in k) or ("/" in k) or ("mouse" in k):
```

Comment:
Let's put all of the substrings we're looking for in a list and then use any() to make this a little more readable/maintainable. Something like
```keys_to_remove = ['+', '>' ...]
if any(substring in k for substring in keys_to_remove):
...
```

Reply:
Completed. Thank you!